doc/specifications/translations/removal.md

```diff
@@ -14,7 +14,7 @@ the existing Versions of a Content Object and is specified as:
  *       a warning before performing it.
  *
  * @throws \eZ\Publish\API\Repository\Exceptions\BadStateException if the specified translation
- *         is the only one Content Object has.
+ *         is the only one Content Object has or it is the main translation.
```

Comment:
well, update phpdoc here on top, drop the exception on initial language check, and as you say add logic to update initial language if we remove it. right?